deploy/cleanup.sh

```diff
@@ -3,54 +3,76 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-set -x
-
 # parameters
-envCode=${1:-"aoi"}
+ENV_CODE=${1:-"aoi"}
+PRE_PROVISIONED_BATCH_ACCOUNT_NAME=${2:-$PRE_PROVISIONED_BATCH_ACCOUNT_NAME}
 
 NO_DELETE_DATA_RESOURCE_GROUP=${NO_DELETE_DATA_RESOURCE_GROUP:-"false"}
-DATA_RESOURCE_GROUP="${envCode}-data-rg"
+DATA_RESOURCE_GROUP="${ENV_CODE}-data-rg"
 
 NO_DELETE_MONITORING_RESOURCE_GROUP=${NO_DELETE_MONITORING_RESOURCE_GROUP:-"false"}
-MONITORING_RESOURCE_GROUP="${envCode}-monitor-rg"
+MONITORING_RESOURCE_GROUP="${ENV_CODE}-monitor-rg"
 
 NO_DELETE_NETWORKING_RESOURCE_GROUP=${NO_DELETE_NETWORKING_RESOURCE_GROUP:-"false"}
-NETWORKING_RESOURCE_GROUP="${envCode}-network-rg"
+NETWORKING_RESOURCE_GROUP="${ENV_CODE}-network-rg"
 
 NO_DELETE_ORCHESTRATION_RESOURCE_GROUP=${NO_DELETE_ORCHESTRATION_RESOURCE_GROUP:-"false"}
-ORCHESTRATION_RESOURCE_GROUP="${envCode}-orc-rg"
+ORCHESTRATION_RESOURCE_GROUP="${ENV_CODE}-orc-rg"
 
 NO_DELETE_PIPELINE_RESOURCE_GROUP=${NO_DELETE_PIPELINE_RESOURCE_GROUP:-"false"}
-PIPELINE_RESOURCE_GROUP="${envCode}-pipeline-rg"
+PIPELINE_RESOURCE_GROUP="${ENV_CODE}-pipeline-rg"
 
 if [[ ${NO_DELETE_DATA_RESOURCE_GROUP} != "true" ]] && [[ ${NO_DELETE_DATA_RESOURCE_GROUP} == "false" ]]; then
     set -x
     az group delete --name ${DATA_RESOURCE_GROUP} --no-wait --yes
+    az deployment sub delete --name ${DATA_RESOURCE_GROUP} --no-wait
```

Comment:
Would this error out if the deployment is not found?

Reply:
yes, but the same will not stop the execution of next statements.